Dockerfile

```
@@ -470,7 +470,7 @@ ARG CGO_ENABLED=1
 ARG GOARCH_aarch64="arm64"
 ARG GOARCH_x86_64="amd64"
 ARG GOARCH_ARCH="GOARCH_${ARCH}"
-ARG CFLAGS="-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-clash-protection"
+ARG CFLAGS="-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-clash-protection"
```

Does it work to just remove `-Wall`?
```suggestion
ARG CFLAGS="-O2 -g -pipe -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-clash-protection"
```

---

`-Werror=format-security` doesn't work on it own, but `-Wformat -Werror=format-security` builds. Should we switch to that instead?

---

Sounds good.